src/routes/ar-io.ts

```diff
@@ -218,7 +223,7 @@ arIoRouter.post(
       const queuedBundle = await system.queueBundle(
         { id, root_tx_id: id } as NormalizedDataItem | PartialJsonTransaction,
         true,
-        true,
+        bypassFilter,
```

Comment:
@dtfiedler I realized after writing this ticket, that passing `bypassFilter: false` will be insufficient to cause filters to apply in the expected way since we have no metadata (tags, owner, etc.) associated with the ID when its queued here. I think what we should do instead is assume this is a base layer TX and queue the TX to be fetched by the `txFetcher` instead. Arguably, this is duplicate functionality (`/ar-io/admin/queue-tx` can accomplish this too), but I think it's probably still worth having the option given that `/ar-io/admin/queue-bundle` is most likely what people will reach for when they want to queue bundles.

Reply:
cc @karlprieb for a sanity check on this.

Reply:
@dtfiedler Still debating this a bit, so I'd say hold off any modifications for now. We could handle this via documentation too.

Reply:
After some Slack discussion, I think we'll go for it. Queue to `txFetcher` instead when `bypassFilter` is set in the request.

Reply:
updated here - [7eb4dba](https://github.com/ar-io/ar-io-node/pull/286/commits/7eb4dbaccf9ef325621b064a3da04751d602d38c)

i put a note that we could alternatively redirect